Symfony/CS/Tests/AbstractIntegrationTest.php

```diff
@@ -270,4 +278,45 @@ protected function getFixersFromConfig($fileName, $config)
 
         return $fixers;
     }
+
+    /**
+     * Parses the '--REQUIREMENTS--' block of a '.test' file and determines requirements.
+     *
+     * @param string $fileName
+     * @param string $config
+     *
+     * @return array
+     */
+    protected function getRequirementsFromConfig($fileName, $config)
+    {
+        $requirements = array('hhvm' => true, 'php' => PHP_VERSION);
```

Comment:
what if I don't set it at all? PHP requires PHP in at least given version, but hhvm is a setting not to require run under HHVM, but to allow run under it, then, the default value is not documented.